File: src/Models/UserStats.ts
import { User } from "./User";

export class UserStats {
    user: User;
    ordersNB: number;
    ordersCoast: number;

    constructor(user: User, ordersNB: number, ordersCoast: number) {
        this.user = user;
        this.ordersNB = ordersNB;
        this.ordersCoast = ordersCoast;
      }
  }
  

File: app/PFE-Back/src/main/java/com/example/PFE/Back/DTO/UserStats.java
package com.example.PFE.Back.DTO;

import com.example.PFE.Back.Model.User;
import com.example.PFE.Back.Model.UserRole;
import jakarta.persistence.criteria.CriteriaBuilder;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UserStats {
   private User user;
    private Integer ordersNB;
    private Double ordersCoast;
}
